frontend/src/components/Item/Metadata/MetadataEditor.vue

```diff
@@ -36,6 +36,14 @@
         v-model="tabName"
         class="pa-2 flex-fill">
         <VWindowItem value="general">
+          <VSelect
+            v-if="contentOptions.length > 1"
```

Comment:
Not sure if this is what you wanted? I guess we want here a disabled selector anyway?
```suggestion
            v-if="contentOptions.length > 0"
```

Can you also explain to me what `contentOptions` are supposed to be? This API still baffles me to this day 😅

Reply:
I actually do not know too lol. There is a lot of weird thing in the API honestly.